rasgotransforms/rasgotransforms/transforms/vlookup/vlookup.yaml

```diff
@@ -0,0 +1,44 @@
+name: vlookup
+tags:
+  - table
+  - join
+  - reshape
+description: |
+  ## Inspired by Excel... a VLookup experience that works in SQL
+
+  ### Required Inputs
+  - Lookup Column: The column to look up in the Lookup Table. Make sure the column is named the same in both tables.
```

Comment:
How soon do you anticipate users wanting to join on columns that don't have the same name? i.e. should we loosen the same name rule and allows for anchor_ & lookup_column args in the event they aren't named the same (which excel supports)? Or do we just push users to a regular join transform at that point?

Reply:
yeah i was wondering the same thing, but i tried to stay true to how vlookup does the lookup and it doesn't have a concept of "mapping" one column name to another (it just goes by position)